tests/aws/services/ec2/test_ec2.py

```diff
@@ -33,6 +39,24 @@ def create(template_name):
             aws_client.ec2.delete_launch_template(LaunchTemplateId=id)
 
 
+@pytest.fixture()
+def create_vpc(aws_client):
+    vpcs = []
+
+    def _create_vpc(
+        CidrBlock: str,
+        TagSpecifications: list[dict],
+    ):
+        vpc = aws_client.ec2.create_vpc(CidrBlock=CidrBlock, TagSpecifications=TagSpecifications)
+        vpcs.append(vpc["Vpc"]["VpcId"])
+        return vpc
+
+    yield _create_vpc
+
+    for vpc_id in vpcs:
+        aws_client.ec2.delete_vpc(VpcId=vpc_id)
```

Comment:
For added cleanup resilience, you can

```suggestion
        try:
            aws_client.ec2.delete_vpc(VpcId=vpc_id)
        except Exception:
            pass
```

Reply:
Cool!